### ISSUE TITLE
is it possible to change the color style?

### ISSUE THREAD

**Issue by garaud** [jonathanslenders/ptpython#45]
Hi,

I see there is `PythonStyle` class at [style.py](https://github.com/jonathanslenders/ptpython/blob/master/ptpython/style.py). For my dark background, the color style is not very readable.

Is there a simple way to configure that? I can try a monkey patch but...well... 

Thanks,
Damien

**Comment by jonathanslenders** [comment]
Currently, there is no easy way yet. But it is coming. Yesterday I started on a way to configure ``ptpython``. Hopefully I can finish and commit that this week-end. The next step is then to support multiple colour schemes. Probably the same way as I did for ``pyvim``.

I'll keep you posted.

**Comment by garaud** [comment]
Nice !! Thank you!
I'm looking forward to it.

**Comment by jonathanslenders** [comment]
So, a simple configuration is already possible: https://github.com/jonathanslenders/ptpython/blob/master/examples/ptpython_config/config.py

Next step is custom colorschemes.

**Comment by garaud** [comment]
Nice. Thank you. I'll update my repo and take a look.

**Comment by jonathanslenders** [comment]
Hi @garaud,

Custom color schemes are possible now.
It's not yet on Pypi. I have to do some testing to be sure it doesn't break anything, but I'll release soon.

Feel free to give it a try already and give me some feedback.
Check out the latest master commit of both ``ptpython`` and ``prompt_toolkit``.
Then check out the following configuration file to see how to add a new color scheme:
https://github.com/jonathanslenders/ptpython/blob/master/examples/ptpython_config/config.py
At runtime, you can choose a colorscheme from the sidebar. (Pressing F2)

Jonathan

**Comment by garaud** [comment]
Nice! I check out prompt_toolkit and ptpython. The F2 sidebar is very useful. I'm trying to get a colorsheme for my dark background theme.

Damien

**Comment by jonathanslenders** [comment]
Hi @garaud,
Yesterday I released a new version of ``ptpython`` that includes color scheme support.
Please tell me when you have some issues.
Cheers.

**Comment by garaud** [comment]
Thanks!! I'll do it (if any).

**Comment by gnudiff** [comment]
I installed ptpython via pip recently (distro Python 2.7 on Ubuntu 16.04) and on my dark background terminal I can't see a lot of the code either -- particulary strings in quotes, anything dark blue, etc. Making my own color scheme seems excessive, there must be some already made?

I found somewhere on internet where somebody had used:
`list(ptpython.style.get_all_styles())`
and I did get a pretty list of styles:

`['manni', 'igor', 'lovelace', 'xcode', 'vim', 'autumn', 'abap', 'vs', 'rrt', 'native', 'perldoc', 'borland', 'arduino', 'tango', 'emacs', 'friendly', 'monokai', 'paraiso-dark', 'colorful', 'murphy', 'bw', 'pastie', 'rainbow_dash', 'algol_nu', 'paraiso-light', 'trac', 'default', 'algol', 'fruity']`

which I am able to use for repl.use_code_colorscheme('pastie') in ~/.ptpython/config.py

In the pip installed version, by the way, there is no code anwyhere in /usr/local/python-2.7/dist/packages/ptpython* which contains any of the names of these styles, although they are available?:
```
$find /usr/local/lib/python2.7/dist-packages/ptpython* -type f | xargs grep pastie
$
```

However, these colorschemes seem to be unavailable for UI itself, which is by default way too dark for using on black background terminal.

I tried to add `repl.use_ui_colorscheme('paraiso-light') `but got an error (probably obviously, but you never know).

Are there any predefined colorschemes for UI? 

Also, it appears when I selected a colorscheme for me in config.py and later in ptpython pressed F3 for history -- the code colorcsheme got completely haywired and didn't come back to the original. But I need to check it more.
